### PR TITLE
Fix broken toggle button by renaming Interactable.dimensions to Interactable.Dimensions

### PR DESCRIPTION
**Pull request by julenka**
## Overview
#6104 had changed Interactable.Dimensions to Interactable.dimensions and correctly marked the field as "FormerlySerializedAs", but Unity did not consistently deserialize the old assets correctly. Go back to the old naming to make sure Unity deserializes properly.

Also update near menu prefabs so the toggle state is correct.

## Changes
- Fixes: #6169

### CODE REVIEW

**Comment by cre8ivepark** [comment]
For the NearMenu update, could you please update the NearMenuExample4x1.prefab instead of the unity scene? Other NearMenu prefab variants will be updated together.

File: Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity [hunk 1]
```diff
@@ -903,6 +903,11 @@ PrefabInstance:
       propertyPath: m_Mesh
       value: 
       objectReference: {fileID: 0}
+    - target: {fileID: 1874729665501627384, guid: 3f1f46cbecbe08e46a303ccfdb5b498a,
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity]: The hand interaction example change is out of place, I'll remove it.

File: Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/NearMenuExamples.unity [hunk 2]
```diff
@@ -594,6 +594,11 @@ PrefabInstance:
       propertyPath: autoFollowAtDistance
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 4669628316120752027, guid: d82e2f09994ad2f4fb43359bfb977f64,
```
Comment (julenka) [inline Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/NearMenuExamples.unity]: I should see if I can avoid these changes by just modifying the toggle features panel prefab

**Comment by julenka** [comment]
> For the NearMenu update, could you please update the NearMenuExample4x1.prefab instead of the unity scene? Other NearMenu prefab variants will be updated together.

Yup

**Pull request merged**